tox.ini

```diff
@@ -124,6 +124,9 @@ skip_install = True
 deps = isort
 commands = /bin/sh -c "isort -c -df -sp setup.cfg -rc synapse tests"
 
+[testenv:check_config]
+commands = {toxinidir}/scripts-dev/lint-config.sh
```

Comment:
Why not just add this to the `check_codestyle` env?

Reply:
I believe my thought process is that we'd have other config-related checks in the future, but you're right in that there's not a need to burn another agent when we have a codestyle one already.